onnx/backend/test/case/node/abs.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np
+
+import onnx
+from .base import Base, expect
+
+
+class Abs(Base):
+
+    @staticmethod
+    def export():
+        node = onnx.helper.make_node(
+            'Abs',
```

Comment:
We really should get in the habit of specifying which version of the operator we are testing ;)